osbs/constants.py

```diff
@@ -102,3 +102,5 @@
 BUILD_TYPE_WORKER = "worker"
 
 ISOLATED_RELEASE_FORMAT = re.compile(r'^\d+\.\d+(\..+)?$')
+
+ANNOTATION_SOURCE_REPO = 'osbs/source_repo'
```

Comment:
We can still use `openshift.io/image.insecureRepository`. `dockerImageRepository` was removed because it causes unwanted behavior. I think we should leverage existing annotations as possible.

Reply:
Yes, just noting that we use a variable name [here](https://github.com/projectatomic/osbs-client/pull/759/files#diff-034b3f8b7a38c1ab83d3dc6cce5f2683R1055) but a string literal [a few lines below that](https://github.com/projectatomic/osbs-client/pull/759/files#diff-034b3f8b7a38c1ab83d3dc6cce5f2683R1057).

Reply:
Ah! Added new commit for it now that I understand your comment